recipe/meta.yaml

```diff
@@ -19,14 +19,14 @@ requirements:
         - automake
         - libtool
         - pkg-config
-        - icu 56.*
+        - icu 56.* 56.*
```

Comment:
NVM. Guess PR ( https://github.com/conda-forge/libxml2-feedstock/pull/11 ) does this.